### PR TITLE
Create rates dictionary for currency

### PR DESCRIPTION
**Pull request by shannon-donahue**
Fixes [COST-2730](https://issues.redhat.com/browse/COST-2730)

Changes proposed in this PR:
* Create a table in the DB for the currency exchange rate dictionary
* Populate the dictionary when the celery task for the currency exchange rates is running

Testing Instructions:
1. Pull in Branch
2. Bring docker up
3. Hit the following endpoint to run celery task: [update_exchange_rates](http://127.0.0.1:5042/api/cost-management/v1/update_exchange_rates/) 
you should see the following output (rates values may be subject to change)
   ```
   "updated_exchange_rates": {
        "USD": 1,
        "AUD": 1.44,
        "CAD": 1.29,
        "CHF": 0.995,
        "CNY": 6.75,
        "DKK": 7.15,
        "EUR": 0.959,
        "GBP": 0.822,
        "HKD": 7.85,
        "JPY": 134.22,
        "NOK": 9.88,
        "NZD": 1.59,
        "SEK": 10.17,
        "SGD": 1.39,
        "ZAR": 16.09
    }
   ```
4. After that hit the following endpoint: [exchange-rates](http://127.0.0.1:8000/api/cost-management/v1/exchange-rates/)
you should see a nested dictionary that can exchange data between all currencies Example format:
```
"AUD": {
        "AUD": 1.0,
        "CAD": 0.8958333333333334,
        "CHF": 0.6909722222222222,
        "CNY": 4.6875,
        "DKK": 4.965277777777779,
        "EUR": 0.6659722222222222,
        "GBP": 0.5708333333333333,
        "HKD": 5.451388888888889,
        "JPY": 93.20833333333334,
        "NOK": 6.861111111111112,
        "NZD": 1.1041666666666667,
        "SEK": 7.0625,
        "SGD": 0.9652777777777778,
        "USD": 0.6944444444444444,
        "ZAR": 11.17361111111111
    },
```

---
Additional Context:

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
be79ce08-09ef-4f94-a430-07e9387983ba

**Review by Fozzywozzybear — APPROVED** [review]
LGTM!

File: koku/api/currency/utils.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    """Build the exchange rates dictionary"""
+    currency_dict = {}
+    base_currency_list = list(rates.keys())
+    base_currency = base_currency_list[index]
+    for code in rates:
+        if base_currency == "USD":
+            currency_dict[code] = rates[code]
+        else:
+            if code == "USD":
+                currency_dict[code] = float(1 / rates[base_currency])
```
Comment (Red-HAP) [inline koku/api/currency/utils.py:22]: float is imprecise. I'd suggest using decimal.Decimal
The math could be different in subsequent runs.

**Review by Red-HAP — CHANGES_REQUESTED** [review]

**Review by lcouzens — APPROVED** [review]
Great job!

**Pull request merged**